sensor_msgs/Image robot_view
---
int32[] bounding_boxes_pixels
